js/src/jsonrpc/interfaces/db.js

```diff
@@ -18,6 +18,7 @@ import { Data } from '../types';
 
 export default {
   getHex: {
+    nodoc: 'Not present in Rust code',
```

Comment:
Think we need to make a call to remove these or move out of the way. (DB is deprecated anyway.) Should just log it, no need to clutter this one.

(Same probably goes for future work, e.g. `shh` - which we may have soon anyway)